test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.test.cpp

```diff
@@ -41,6 +41,42 @@ class OpenSpaceToolkit_Physics_Coordinate_Frame_Providers_IERS_Manager : public
     {
         this->bulletinA_ = BulletinA::Load(bulletinAFile_);
         this->finals2000A_ = Finals2000A::Load(finals2000AFile_);
+
+        // cache current directory environment variables
+        localRepositoryPath = std::getenv(localRepositoryVarName);
+        fullDataPath = std::getenv(fullDataVarName);
+        modeValue = std::getenv(modeVarName);
+    }
+
+    virtual void TearDown()
+    {
+        // reset cached environment variables
+        if (fullDataPath)
+        {
+            setenv(fullDataVarName, fullDataPath, true);
```

Comment:
`std::setenv` no? and same for `unsetenv`?

Reply:
It's not in the Cpp standard, so likely not:
https://stackoverflow.com/questions/30292642/c-standard-library-stdsetenv-vs-setenv